ext/build_defaults.yaml

```diff
@@ -141,6 +141,8 @@ platform_repos:
     repo_location: repos/solaris/11/**/*.i386.p5p
   - name: solaris-11-sparc
     repo_location: repos/solaris/11/**/*.sparc.p5p
+  - name: windows-2012-x86
+  - name: windows-2012-x64
```

Comment:
So, while we do only build for these two platforms (and we only ship two packages for windows, an x86 and an x64 version) we test those two packages on several different platforms.

In general, you can't use the platform_repos list as the list of platforms to test, as they are different